app/views/cart/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>Cart</h1>
+
+<% @cart.items.each do |item, quantity| %>
+  <section id="cart-item-<%= item.id %>">
+    <ul>
+      <li><%= link_to item.name, item_path(item) %></li>
+      <li><%= link_to image_tag(item.image_url, alt: "Image of #{item.name}"), item_path(item) %></li>
+      <li><%= item.user.name %></li>
+      <li>$<%= item.price.round(2) %></li>
+      <li>Quantity: <%= quantity %></li>
+      <li>Subtotal: <%= item.subtotal(quantity).round(2) %></li>
```

Comment:
Issue later to have 2 digits after the decimal if we use this method and the number ends in zero